appimage/toolchain.bzl

```diff
@@ -0,0 +1,13 @@
+""" Toolchain for the AppImage runtime """
+
+def _appimage_toolchain_impl(ctx):
+    return [platform_common.ToolchainInfo(
```

Comment:
I wonder if it would make sense to add the Python "tool" program (it really needs a better name, maybe mkappimage or so) into the toolchain? It should work without, and I'm not sure if there are any drawbacks to keeping it outside the toolchain. Maybe you have some opinion on that?

Reply:
Since the `tool` isn't architecture dependent, I'm inclined to leave it out of the target specific toolchain